src/App.jsx

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+import useEnigma from './useEnigma';
+import KPI from './KPI';
+import useQlikConnect from './useQlikConnector';
+
+const Wrapper = styled.main`
+  display: flex;
+  flex-direction: row;
+`;
+
+export default function App() {
+  // Define app ID
+  const appId = '7b227df8-8be3-4835-9f4d-617cf47e9437';
+
+  // Custom hook
+  const enigma = useQlikConnect(appId);
+
+  // Table data for total revenue
+  const revenueTable = useEnigma(enigma.doc, 'tWJJyZ').qlikData;
+
+  // Table data for total expenses
+  const expensesTable = useEnigma(enigma.doc, 'eMsVVT').qlikData;
+
+  // Table data for total profit
+  let profitTable = useEnigma(enigma.doc, 'xWWjCN').qlikData;
+
+ 
+  if (profitTable) {
+    let profits = [];
+
+    // Calculate profit for each quarter
+    profitTable.forEach((row) =>
```

Comment:
FYI: There's a way that you can do the addition with one iterator, instead of having a separate forEach and the reduce.
No change needed here, but msg me if you want me to send you an example.

Reply:
```
let totalProfit = 0

profitTable.forEach((row) => {
  const rowRevenue = row[1].qNum
  const rowExpense = row[2].qNum
  const rowProfit = rowRevenue - rowExpense
  totalProfit += rowProfit // if linter doesn't like this you can do totalProfit = totalProfit + rowProfit
})
```

Reply:
```
    const totalProfit = profitTable.reduce((prevValue, row) => {
      const rowRevenue = row[1].qNum
      const rowExpense = row[2].qNum
      const rowProfit = rowRevenue - rowExpense
      return prevValue += rowProfit
    }, 0) // the 0 is the initial value, i.e. when you hit the first row in profitTable array prevValue starts off as initialValue
```